static_src/app.jsx

```diff
@@ -53,7 +53,10 @@ export default class App extends React.Component {
 
     if (this.state.isLoggedIn) {
       content = this.props.children;
-      sidebar = <Nav initialCurrentOrgGuid={ this.props.currentOrgGuid } />;
+      sidebar = <Nav
+        initialCurrentOrgGuid={ this.props.currentOrgGuid }
+        initialSpaceGuid={ this.props.initialSpaceGuid }
```

Comment:
to be consistent with the naming, should this be `initialSpaceGuid={ this.props.currentSpaceGuid }`